CHANGELOG.md

```diff
@@ -3,6 +3,29 @@
 ### master
 [Full Changelog](https://github.com/parse-community/Parse-SDK-JS/compare/2.11.0...master)
```

Comment:
2.11.0...master should be 2.12.0...master

Reply:
Done